dc/council/code/titles/47/sections/47-2711.xml

```diff
@@ -5,25 +5,25 @@
-      <text>A refund of permit fee shall be made as follows:</text>
-      <text>When no work has been done under authority of permit the fee in excess of the cost of inspection to verify no work having been done, based on $10 per inspector hour, the cost of any engineering examination time previously devoted to approval of plans, based on $15 per hour, plus $14 administrative costs of “issuance and refund”, shall be refunded  14.00</text>
-      <text>When work authorized by permit has been only partially done and when the District is satisfied that no more work will be done under the permit, the fee in excess of the cost of any engineering plans examination based on $15 per hour, inspections costs based on $10 per hour, plus $14 administrative costs of “issuance and refund”, shall be refunded  14.00</text>
-      <text>Provided: That request for refund shall be made within 6 months from date of issuance and the permit and receipt are returned to the Permit Branch.</text>
-      <text>The penalty for a permit to abate notice of doing work without a permit shall be 50% of the fee.</text>
-      <text>No permit fee shall be charged when supported by evidence indicating that the applicant is under contract or subcontract to perform the following:</text>
-      <text>Work done exclusively for the District of Columbia.</text>
-      <text>Work done under contract for the District.</text>
-    </para>
+    <text>
+      <center>Riparian Permits Schedule</center>
+    </text>
+    <text>Fees required by the Rules and Regulations for the Government of Riparian Rights and Water Privileges in the District of Columbia.</text>
+    <text>Fees for permits to fill or dredge, construct, reconstruct or repair any structure shall be as follows:</text>
+    <text>
+      <em>Fee</em>
+    </text>
+    <text>Work costing up to $500  $ 9.00</text>
+    <text>Work costing from $501 to $1,000   14.00</text>
+    <text>Each additional $1,000 of increased cost   14.00</text>
+    <text>A refund of permit fee shall be made as follows:</text>
+    <text>When no work has been done under authority of permit the fee in excess of the cost of inspection to verify no work having been done, based on $10 per inspector hour, the cost of any engineering examination time previously devoted to approval of plans, based on $15 per hour, plus $14 administrative costs of “issuance and refund”, shall be refunded  14.00</text>
+    <text>When work authorized by permit has been only partially done and when the District is satisfied that no more work will be done under the permit, the fee in excess of the cost of any engineering plans examination based on $15 per hour, inspections costs based on $10 per hour, plus $14 administrative costs of “issuance and refund”, shall be refunded  14.00</text>
+    <text>Provided: That request for refund shall be made within 6 months from date of issuance and the permit and receipt are returned to the Permit Branch.</text>
```

Comment:
The "Provided" is italicized. (Division VIII, pg 1998). Change likely not needed for validation.